components/progress-bar/src/ProgressBar.js

```diff
@@ -5,90 +5,91 @@ import PropTypes from 'prop-types';
+ * Use when loading content or to indicate how far along
+ * the user is in a journey.
  */
-export const ProgressBar = ({ value, isSkinny, ...props }) => {
-	const { progressBar } = useTheme();
-
-	const valueRound = Math.round(value);
-
-	const size = isSkinny ? 'skinny' : 'default';
-
-	const style = {
-		// Common styling
-		common: {
-			height: progressBar.size[size].height,
-			marginBottom: progressBar.marginBottom,
-			overflow: 'hidden',
-			backgroundColor: 'white',
-			borderRadius: progressBar.size[size].height,
-			border: `${progressBar.borderWidth} solid ${progressBar.borderColor}`,
-			padding: progressBar.padding,
-			position: 'relative',
+export const ProgressBar = ({ value, skinny, ...props }) => {
+	const { COLORS } = useTheme();
 
-			'::after': {
-				display: isSkinny && 'none',
-				content: '"0%"',
-				position: 'absolute',
-				left: '1rem',
-				top: 0,
-				height: '100%',
-				color: '#575f65',
-				fontSize: '1.4rem',
-				fontWeight: 700,
-				zIndex: 1,
-			},
-		},
+	const roundedValue = round(value);
 
-		// Bar styling
-		bar: {
-			display: 'flex',
-			justifyContent: 'flex-end',
-			alignItems: 'center',
-			position: 'relative',
-			float: 'left',
-			width: 0,
-			height: '100%',
-			fontSize: progressBar.fontSize,
-			fontWeight: progressBar.fontWeight,
-			lineHeight: progressBar.lineHeight,
-			color: progressBar.color,
-			textAlign: 'right',
-			backgroundColor: progressBar.backgroundColor,
-			borderRadius: progressBar.size[size].height,
-			zIndex: 2,
-			overflow: 'hidden',
-			transition: 'width .6s ease',
-
-			'@media print': {
-				backgroundColor: 'black !important',
+	return (
+		<div
+			css={{
+				height: skinny ? '0.625rem' : '1.5rem',
+				marginBottom: '1.3125rem',
+				overflow: 'hidden',
+				backgroundColor: '#fff',
+				borderRadius: skinny ? '0.625rem' : '1.5rem',
+				border: `1px solid ${COLORS.border}`,
+				padding: '0.0625rem',
+				position: 'relative',
 
-				span: {
-					color: 'white !important',
-					backgroundColor: 'black !important',
+				'::after': {
+					display: skinny && 'none',
+					content: '"0%"',
+					position: 'absolute',
+					left: '0.625rem',
+					top: 0,
+					height: '100%',
+					color: '#575f65',
+					fontSize: '0.875rem',
+					fontWeight: 700,
+					zIndex: 1,
 				},
-			},
-		},
-	};
-
-	return (
-		<div css={style.common} {...props}>
+			}}
+			{...props}
+		>
 			<div
-				css={style.bar}
-				style={{ width: `${valueRound}%` }}
+				css={{
+					display: 'flex',
+					justifyContent: 'flex-end',
+					alignItems: 'center',
+					position: 'relative',
+					float: 'left',
+					width: 0,
+					height: '100%',
+					fontSize: '0.875rem',
+					fontWeight: 700,
+					lineHeight: '1.25rem',
+					color: '#fff',
+					textAlign: 'right',
+					backgroundColor: COLORS.hero,
+					borderRadius: skinny ? '0.625rem' : '1.5rem',
+					zIndex: 2,
+					overflow: 'hidden',
+					transition: 'width .6s ease',
+
```

Comment:
we concerned about line breaks inside style blocks? Whitespace or no whitespace? (@simonswiss I see you don't like them?)